### ISSUE TITLE
I18N support 

### ISSUE THREAD

**Issue by johnfelipe** [URUWorks/TeroSubtitler#173]
I want to contribute translation to Spanish a d Italian if u like...

**Comment by URUWorks** [comment]
Of course, you can help with the translation you like and that is not currently available.
